src/main/java/uk/gov/ch/developer/docs/controller/developer/UserCallbackController.java

```diff
@@ -35,6 +40,19 @@ public String getCallback(@RequestParam("state") String state,
             // Nonces
         }
 
+        WebClient webClient = WebClient.create();
+
+        webClient.post().uri(URI.create(identityProvider.getTokenUrl()))
+                .contentType(MediaType.APPLICATION_FORM_URLENCODED)
+                .accept(MediaType.APPLICATION_JSON)
+                .body(BodyInserters.fromPublisher(Mono.just("code=" + code + "&client_id="
+                        + identityProvider.getClientId() + "&client_secret="
+                        + identityProvider.getClientSecret() + "&redirect_uri="
+                        + identityProvider.getRedirectUri() + "&grant_type=authorization_code"),
+                        String.class))
+                .retrieve().bodyToMono(OAuthToken.class)
+                .subscribe(s -> LOGGER.trace("Access Token " + s.getAccess_token()));
```

Comment:
Seems like a bad idea to log access tokens.

Reply:
This is to prove that we have an access token so it can be tested, it will be removed in the next ticket.

Reply:
Please add a //TODO to remove the logging line later.

Reply:
If these are for debugging while the code is being built, it might be good to mark this with a //TODO remove the logging when we clean up.